spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -453,7 +453,8 @@ class CythonSH(PythonSH):
 #==============================================================================
 class EnamlSH(PythonSH):
     """Enaml Syntax Highlighter"""
-    ADDITIONAL_KEYWORDS = ["enamldef", "template", "attr", "event", "const", "alias"]
+    ADDITIONAL_KEYWORDS = ["enamldef", "template", "attr", "event", "const", "alias",
+                           "func"]

```

Comment:
func is used to declare a func inside an enamldef block.

```python
enamldef MyWidget(PushButton):
    func hello_world():
        print 'Hello world'
    clicked::
         hello_world()
```
Some functions can also be declared as declarative and overridden in the declaration, but I don't remember the syntax. My point here is just that as the language supports it it makes sense for the highlighter to support it too.